chef/cookbooks/network/recipes/default.rb

```diff
@@ -566,6 +572,13 @@ def get_datapath_id_for_ovsbridge(bridge)
         pre_up_script: pre_up_script
       })
     end
+    # Mark all changed interfaces as up, so wicked will keep them that way.
+    bash "wicked ifup #{nic.name}" do
+      code "wicked ifup #{nic.name}"
+      action :nothing
+      subscribes :run, "template[/etc/sysconfig/network/ifcfg-#{nic.name}]", :immediately
+      only_if { nic_ifcfg.updated_by_last_action? }
```

Comment:
Why is this needed? The ```subscribes``` already makes sure that the resource is only executed when the template is changed.